drivers/network/tcpip/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ set(LWIP_INCLUDE_DIRS
     ${LWIP_DIR}/src/include
     ${CMAKE_CURRENT_SOURCE_DIR}/include/lwip
 )
-add_subdirectory(lwip)
+include(${LWIP_DIR}/src/Filelists.cmake)
+add_dependencies(lwipcore xdk)
```

Comment:
Is this add_dependencies needed because in a sense, this is needed in the 3rdparty Filelists.cmake?

Reply:
lwIP sources (lwipcore library) include drivers/network/tcpip/lwip/src/include/lwip/arch.h, which includes drivers/network/tcpip/lwip/src/include/lwip/arch.h, which includes drivers/network/tcpip/include/lwip/arch/cc.h, which includes wdm.h.
So yes, lwipcore library depends of xdk